### DISCUSSION TITLE
Changing how leap treats the first item it finds

### DISCUSSION THREAD

**Discussion by Wiebesiek** [ggandor/leap.nvim#158] [Q&A] [answered]
Is there a way to adjust how leap treats the first item? I think I would prefer that the first item is treated like any other item that is found. i.e. has a label, needs the label pressed to jump to it.

**Comment by ggandor** [comment, marked as answer]
See FAQ, or `:help leap.opts.safe_labels`. (Set `opts.safe_labels` to `{}`.)

**Reply by Wiebesiek** [reply]
Thanks for this! This feels more comfortable :)
